pkg/kube/translation/plugin.go

```diff
@@ -115,6 +116,9 @@ func (t *translator) translateConsumerBasicAuthPlugin(consumerNamespace string,
 
 func (t *translator) translateConsumerJwtAuthPlugin(consumerNamespace string, cfg *configv2beta3.ApisixConsumerJwtAuth) (*apisixv1.JwtAuthConsumerConfig, error) {
 	if cfg.Value != nil {
+		if cfg.Value.Exp < 0 {
```

Comment:
https://github.com/apache/apisix/blob/ab7a7675d2a1abb1f0299a8fd44e5f80435c2158/apisix/plugins/jwt-auth.lua#L66

the minimum is 1